trezorlib/tests/device_tests/test_msg_signtx_komodo.py

```diff
@@ -0,0 +1,170 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib import btc, messages as proto
+from trezorlib.tools import parse_path
+
+from ..support.tx_cache import tx_cache
+from .common import TrezorTest
+
+TX_API = tx_cache("Komodo Mainnet")
```

Comment:
Correct network name is `"Komodo"` not `"Komodo Mainnet"`. I doubt you have tried to run these tests, otherwise you have found this issue immediately.